docs/smart-contracts/creator-tools/ZoraNFTCreator.mdx

```diff
@@ -1,12 +1,14 @@
 import EvD from "../../../static/img/EvD.png"
 
-##### A factory that deploys new NFT contracts
+##### A factory that deploys new 721 NFT contracts
+
+> Note: 721 NFTs are ***only*** supported on the contract and API level; users are ***no longer*** able to create or update 721 NFTs on the UI level.
```

Comment:
i would say:
721 NFTs can only be created and managed on the contract and API level, users are ***no longer*** able to create or update 721 NFTs on the zora.co UI